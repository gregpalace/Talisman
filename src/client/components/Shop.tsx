import React from 'react';

import ShopKeepSprite from '../../assets/tiles/shop-keep.png';

import '../../stylesheets/shop.scss';

const ShopKeep = () => {
  return (
    <div
      className='shop-keep-animated'
      style={{ backgroundImage: `url(${ShopKeepSprite})` }}
    />
  );
};

export default ShopKeep;
