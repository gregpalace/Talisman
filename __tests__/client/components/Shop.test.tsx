import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Shop from '../../../src/client/components/Shop';

describe('React Unit Tests', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<Shop />);
  });

  describe('<Shop />', () => {
    it('should render correctly', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
