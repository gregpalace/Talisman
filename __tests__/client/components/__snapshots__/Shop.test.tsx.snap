// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Unit Tests <Shop /> should render correctly 1`] = `undefined`;
